yb-voyager/src/srcdb/postgres.go

```diff
@@ -669,3 +669,31 @@ func (pg *PostgreSQL) GetNonPKTables() ([]string, error) {
 	}
 	return nonPKTables, nil
 }
+
+func (pg *PostgreSQL) GetTablesWithReplicaIdentityNotFull(tableList []*sqlname.SourceName) ([]string, error) {
+	var tablesWithReplicaIdentityNotFull []string
+	var qualifiedTableNames []string
+	for _, table := range tableList {
+		qualifiedTableNames = append(qualifiedTableNames, fmt.Sprintf("'%s'", table.Qualified.MinQuoted))
```

Comment:
pls check that this works for case-sensitive tables as well. "FoO" for example. Check what the value for this table is in c.relname. Is it with quotes, without quotes? I'm assuming it should be without quotes, in which case MinQuoted would be wrong?

Reply:
Leaving it out now since the current live migration doesnt support case sensitive tables